tests/unit_tests/query_handler_runner/test_python_query_handler_runner.py

```diff
@@ -199,13 +199,13 @@ def test_continue_finish(temporary_schema_name, top_level_query_handler_context)
         sql_executor.queries == [
             cleandoc(
                 f"""
-                CREATE OR REPLACE VIEW "{temporary_schema_name}"."temp_db_object_2_1" AS\x20
+                CREATE OR REPLACE VIEW "{temporary_schema_name}"."temp_db_object_2_1" AS
                 SELECT 1 as "a";
                 """
             ),
             cleandoc(
                 f"""
-                SELECT\x20
+                SELECT
```

Comment:
did you find the space? very good.

Reply:
:rofl: - I admit it was a failing test forcing me to remove the `\x20`. 
I assumed somebody else applied a change in the meantime ?